src/test/resources/features/atualizacaoStatusApostas.feature

```diff
@@ -0,0 +1,44 @@
+#language: pt
+
+@AtualizacaoStatusAposta
+Funcionalidade: Realizar atualização do status de apostas disponíveis no banco de dados
+
+  Contexto:
+    Dado que existam as seguintes apostas cadastradas no banco de dados
+      | idt_bet | num_odd | flg_selected | dat_created | dat_updated | des_bet                     | bet_status |
+      | 1       | 2.13    | 0            | 2024-08-15  | 2024-08-15  | Vitória do São Paulo        | PENDING    |
+      | 2       | 1.32    | 0            | 2024-08-15  | 2024-08-15  | Vitória do Flamengo         | PENDING    |
+      | 3       | 1.45    | 0            | 2024-08-15  | 2024-08-15  | Vitória do Palmeiras        | PENDING    |
+      | 4       | 4.12    | 1            | 2024-08-16  | 2024-08-21  | Vitória do Corinthians      | GREEN      |
+      | 5       | 1.19    | 0            | 2024-08-17  | 2024-08-17  | Vitória do Real Madrid      | RED        |
+      | 6       | 1.56    | 1            | 2024-08-17  | 2024-08-18  | Vitória do Botafogo         | GREEN      |
+      | 7       | 2.67    | 0            | 2024-08-18  | 2024-08-18  | Vitória do Bahia            | RED        |
+      | 8       | 4.90    | 1            | 2024-08-18  | 2024-08-19  | Vitória do Vitória          | PENDING    |
+      | 9       | 2.91    | 0            | 2024-08-20  | 2024-08-21  | Vitória do Grêmio           | PENDING    |
+      | 10      | 3.02    | 1            | 2024-08-20  | 2024-08-20  | Vitória do Internacional    | GREEN      |
+      | 11      | 2.15    | 0            | 2024-08-20  | 2024-08-22  | Vitória do Cruzeiro         | RED        |
+      | 12      | 2.20    | 1            | 2024-08-20  | 2024-08-20  | Vitória do Atlético Mineiro | RED        |
+
+  @AtualizacaoStatusApostaSucesso
+  Cenario: Atualização de status de aposta com sucesso
+    Quando uma requisição de atualização de aposta for realizada com status "GREEN" e idt_bet 1
+    Entao o serviço de atualização deve retornar o status code 200
```

Comment:
Tome cuidado com o reuso de steps, esse step está nos arquivos: "atualizacaoApostas.feature, atualizacaoStatusApostas.feature"

Nesse caso acredito que não haverá problema, pois o comportamento esperado e o objeto provavelmente será o mesmo. Mas tome ponto:

O comportamento esperado for diferente: Se você tiver o mesmo step, mas espera um comportamento diferente em diferentes cenários ou features, então, nesse caso, você precisaria diferenciar os steps.

Confusão na Manutenção: Se os steps são idênticos, mas em diferentes contextos ou funcionalidades, pode ser difícil, em termos de manutenção, identificar rapidamente qual step está sendo executado e qual lógica ele deve seguir. Nesse caso, seria útil tornar os steps mais específicos.